softwarereview_intro.Rmd

```diff
@@ -44,7 +44,7 @@ Furthermore, authors and reviewers have the ability to contact privately the edi
 
 - Your package README will feature a peer-review badge linking to the software review thread.
 - Your package will get a [`docs.ropensci.org` docs website](#rodocsci) that you can link from DESCRIPTION.
-- Your package repo will be transferred to the rOpenSci organization.
+- If desired, your package repo may be transferred to the rOpenSci organization.
```

Comment:
For PT translation https://github.com/ropensci/dev_guide/pull/852/files#r2018747592

Reply:
Hi @maelle ! This is updated here: https://github.com/ropensci/dev_guide/pull/822/files > softwarereview_intro.pt.Rmd > line 47.

Reply:
Thank you!!